pygmt/tests/test_xyz2grd.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 import pytest
 import xarray as xr
-from pygmt import grdinfo, xyz2grd
+from pygmt import load_dataarray, xyz2grd
 from pygmt.datasets import load_sample_bathymetry
```

Comment:
```suggestion
from pygmt.datasets import load_sample_data
```
I think you should use ``load_sample_data(name='bathymetry')`` rather than load_sample_bathymetry since the latter will be deprecated.